spec/unit/resource/action_items_spec.rb

```diff
@@ -32,6 +32,16 @@
       expect(resource.action_items.first.html_class).to eq("action_item test")
     end
 
+    it 'should be ordered by priority' do
+      resource.add_action_item :first, priority: 0 do
+        # Empty ...
```

Comment:
please also add one action item without priority to this example for testing default priority